docs/docs/python/overview/pex.mdx

```diff
@@ -67,7 +67,27 @@ pex_binary(
 )
 ```
 
-You can generate the JSON content for these files by installing the [Pex](https://github.com/pex-tool/pex) command-line tool on the target platform and running `pex3 interpreter inspect --markers --tags` against the appropriate interpreter. You can run `pex3 interpreter inspect --help` for more options, and in particular for how to select the desired target interpreter.
+### Generating the `complete_platforms` file
+
+The core command for generating a `complete_platforms` file looks like this (see [pex docs](https://docs.pex-tool.org/buildingpex.html#complete-platform)):
+
+```sh
+pip install pex && pex3 interpreter inspect --markers --tags --indent=2
+```
+
+You can run `pex3 interpreter inspect --help` for more options, and in particular for how to select the desired target interpreter.
+
+You can run this command in a few different ways:
+
+1. Manually/interactively in your target environment
+2. In a Docker container
+
+    If you have a Docker image corresponding to your target environment, you can start a container with that image and run the PEX tool inside it to generate the file:
+
+    ```sh
+    docker run --entrypoint='/bin/sh' -it --rm <docker_repo>:<image_tag> -c "python -m pip install --target=/tmp/pex pex >/dev/null 2>&1 && PYTHONPATH=/tmp/pex/bin/pex3 interpreter inspect --markers --tags --indent=2" > complete_platforms.json
+    ```
+3. Inside a Python process (e.g. [AWS Lambda](../integrations/aws-lambda.mdx#specifying-a-runtime-explicitly))
```

Comment:
(Once we add GCF content, reminder to add a link here too.)

Reply:
Yup, absolutely. I've just been procrastinating on GCF because I need to actually make a cloud function that outputs the `complete_platforms` file and try it out. Once I actually finish that I'll update all GCF docs.

Reply:
Ah I see. It would be fine by me to leave the specific example out of the GCF docs for now (and do it in a docs-only follow-up) to keep this moving forward, and even solicit someone else to write it if you don't use GCF/aren't familiar with it. (I'm not, so I can't really help 😅 )

Reply:
Made the function for GCF + included in the docs!